spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/SparkDistributionAndOrderingUtil.java

```diff
@@ -146,14 +146,14 @@ private static Distribution buildCopyOnWriteDeleteUpdateDistribution(Table table
 
   public static Distribution buildPositionDeltaDistribution(Table table, Command command,
                                                             DistributionMode distributionMode) {
-    if (command == DELETE) {
-      return positionDeleteDistribution(distributionMode);
+    if (command == DELETE || command == UPDATE) {
+      return buildPositionDeleteUpdateDistribution(distributionMode);
     } else {
       throw new IllegalArgumentException("Only position deletes are currently supported");
```

Comment:
[minor] can update the exception message here as well, to include `update` command: 

```suggestion
      throw new IllegalArgumentException("Only position deletes and updates are currently supported");
```

Reply:
Good catch, let me fix it.